### ISSUE TITLE
Henry Upton

### ISSUE THREAD

**Issue by henry-upton** [henry-upton/henry-upton#10]
https://henryupton.com/reviews/hard-boiled-wonderland-and-the-end-of-the-world/

**Comment by DestinedHorizon** [comment]
I agree that some parts of the story are rather difficult to believe, but all the while I would have enjoyed reading this book as it is quite different from the norm. Being able to experience two worlds in a single book sounds like a fun adventure.
